Sources/SwiftFusion/Geometry/Pose3.swift

```diff
@@ -232,7 +210,8 @@ extension Pose3Coordinate: ManifoldCoordinate {
     // simplified with Mathematica, and multiplying in T to avoid matrix math
     let Tan = tan(0.5 * t)
     let WT = matmul(W, T.tensor.reshaped(to: [3, 1]))
-    let u = T.tensor.reshaped(to: [3, 1]) - (0.5 * t) * WT + (1 - t / (2 * Tan)) * matmul(W, WT)
+    let u: Tensor<Double> =
+      T.tensor.reshaped(to: [3, 1]) - (0.5 * t) * WT + (1 - t / (2 * Tan)) * matmul(W, WT)
```

Comment:
BTW, skew(w) * v = just cross(w,v), of course.

Reply:
I'll leave this unchanged because I haven't taken time to understand it.

The reason I have a change here is that I needed to add `: Tensor<Double>` type annotation to deconfuse the typechecker. (The `LieGroup` `*` confused it a bit and it couldn't figure out which `*` was happening here.)